### ISSUE TITLE
Wavedrom 1.4.2 MacOS "is from an unidentified developer"

### ISSUE THREAD

**Issue by thomasrussellmurphy** [wavedrom/wavedrom.github.io#25]
Version 1.4.2 of Wavedrom for MacOS "is from an unidentified developer," failing the MacOS GateKeeper signed software check. I encountered this on OS 10.11.6.

It would be helpful to have an authorized developer sign the MacOS application to avoid being blocked by GateKeeper.

**Comment by drom** [comment]
anybody have idea how to sign MacOS apps? on linux?

**Comment by thomasrussellmurphy** [comment]
Apple provides [Developer ID and Gatekeepr](https://developer.apple.com/developer-id/). Signing shouldn't be a problem on Linux because there isn't a centralized authority trying to restrict code execution.

**Comment by drom** [comment]
@thomasrussellmurphy thank you for the link. I see `codesign` CLI tool they recommend using for the process. Looks like it is proprietary tool running on OSX platform only. Do I miss something?
I have found https://github.com/saucelabs/isign that claim to do it on Linux. Have to try.

**Comment by thomasrussellmurphy** [comment]
Yes, the OSX code-signing from my link is an Apple-proprietary system. The `isign` tool appears to be for the mobile iOS platform apps rather than desktop OSX. I've never developed a (signed) OSX application, so I'm not familiar with the process.
